core/src/avm1/activation.rs

```diff
@@ -2879,6 +2889,11 @@ impl<'a, 'gc> Activation<'a, 'gc> {
         }
     }
 
+    /// Tries to resolve a level by ID. Returns None if it does not exist.
+    pub fn try_resolve_level(&mut self, level_id: i32) -> Option<DisplayObject<'gc>> {
```

Comment:
nit: AFAIK `try_F()` generally means "same as `F()`, but returns Result/Option instead of panic" - at least that's how I understand it from stdlib? So maybe some other name would be slightly better.

Maybe `try_resolve_level` -> `get_level()` and  `resolve_level` -> `get_or_create_level()`?

Reply:
Good shout, renamed!